.cirrus.yml

```diff
@@ -1,547 +1,499 @@
         memory: 12G
-      test_all_script:
+      script:
         - dart --enable-asserts dev\bots\test.dart
-    - name: integration_tests-windows
-      env:
-        SHARD: integration_tests
+
+    # TODO(ianh): integration_tests_gradle1-windows requires https://github.com/flutter/flutter/issues/36311
+
+    # TODO(ianh): integration_tests_gradle2-windows requires https://github.com/flutter/flutter/issues/36311
+
+    - name: integration_tests_misc-windows
       container:
         cpu: 4
         memory: 12G
-      test_all_script:
+      env:
+        SHARD: integration_tests
+        SUBSHARD: misc
+      script:
         - dart --enable-asserts dev\bots\test.dart
+
+    # TODO(ianh): add_to_app-windows
+
     - name: customer_testing-windows
-      test_script:
+      script:
         - CMD /S /C "IF EXIST "bin\cache\pkg\tests\" RMDIR /S /Q bin\cache\pkg\tests"
         - git clone https://github.com/flutter/tests.git bin\cache\pkg\tests
         - dart --enable-asserts dev\customer_testing\run_tests.dart --skip-on-fetch-failure --skip-template bin/cache/pkg/tests/registry/*.test
 
+# MACOS SHARDS
 task:
-  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
-  name: deploy_gallery-macos
-  pub_cache:
-    folder: ~/.pub-cache
-  depends_on:
-    - analyze
-    - build_tests-macos
-  env:
-    # Name the SDK directory to include a space so that we constantly
-    # test path names with spaces in them.
-    CIRRUS_WORKING_DIR: "/tmp/flutter sdk"
-    SHARD: deploy_gallery
-    # Apple Fastlane password.
-    FASTLANE_PASSWORD: ENCRYPTED[4b1f0b8d52874e9de965acd46c79743f3b81f3a513614179b9be7cf53dc8258753e257bdadb11a298ee455259df21865]
-    # Private repo for publishing certificates.
-    PUBLISHING_MATCH_CERTIFICATE_REPO: ENCRYPTED[3c0e78877d933fc80107aa6f3790fd1cf927250b852d6cb53202be696b4903ed8ca839b809626aaf18050bf7e436fab7]
-    PUBLISHING_MATCH_REPO_TOKEN: ENCRYPTED[3d1230b744c6ed6c788a91bec741b769401dbcd426b18f9af8080bfeefdfc21913ca4047980c5b5b7ce823f12e7b6b19]
-    # Apple Certificates Match Passphrase
-    MATCH_PASSWORD: ENCRYPTED[db07f252234397090e3ec59152d9ec1831f5ecd0ef97d247b1dca757bbb9ef9b7c832a39bce2caf1949ccdf097e59a73]
   osx_instance:
-    image: mojave-xcode-10.1
-  # occasionally the clock on these machines is out of sync
-  # with the actual time - this should help to verify
-  print_date_script:
-    - date
-  install_cocoapods_script:
-    - sudo gem install cocoapods
-  git_fetch_script:
-    - git clean -xfd
-    - git fetch origin
-    - git fetch origin master # To set FETCH_HEAD
-  setup_script:
-    - bin/flutter config --no-analytics
-    - bin/flutter update-packages
-  test_all_script:
-    - ./dev/bots/deploy_gallery.sh
-
-task:
-  use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
-  osx_instance:
-    image: mojave-xcode-10.1
-  depends_on:
-    - analyze
+    image: mojave-xcode-10.2-flutter # see https://cirrus-ci.org/guide/macOS/ for list of images (we should update regularly)
```

Comment:
@fkorotkov - we always want to use the latest stable version of Xcode available.  Is it possible to have an image named something like `mojave-xcode-stable-flutter`?

Reply:
Great idea for an alias. Please create an feature request: https://github.com/cirruslabs/cirrus-ci-docs/issues/new?assignees=&labels=feature&template=feature_request.md

Reply:
Done